components/editor/ai-integration/integration-with-inline-prompt.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Integration with Inline Prompt
+page_title: Editor Integration with Inline Prompt
+description: Integration of the Inline Prompt in the Editor for Blazor.
+slug: editor-inline-prompt-integration
+tags: telerik,blazor,editor,ai,ai integration,Inline Prompt
+published: True
+position: 3
+---
+
+# Editor Integration with Inline Prompt
+
+The Editor provides a built-in integration with an Inline Prompt component to help users add AI-driven suggestions, completions, and assistance to their Editor.
```

Comment:
"Inline Prompt" with capital letters looks like some official component. Perhaps remove the capitalization at least from "inline"

```suggestion
The Editor provides a built-in integration with an inline Prompt component to help users add AI-driven suggestions and completions to their Editor.
```